config/config.go

```diff
@@ -240,10 +240,21 @@ type Privacy struct {
 	LMT  LMT
 }
 
+type VendorListFetcher struct {
+	// Max concurrent request for downloading the latest version of the vendor list from a specific major version.
+	// If it is 0 or negative, then means there is no limit for the max concurrency.
+	MaxConcurrencyInitFetchLatestVersion int `mapstructure:"max_concurrency_init_fetch_latest_version"`
+
+	// Max concurrent request for downloading every specific version of the vendor list from its first version to its latest version.
+	// If it is 0 or negative, then means there is no limit for the max concurrency.
+	MaxConcurrencyInitFetchSpecificVersion int `mapstructure:"max_concurrency_init_fetch_specific_version"`
+}
+
 type GDPR struct {
-	Enabled                 bool         `mapstructure:"enabled"`
-	HostVendorID            int          `mapstructure:"host_vendor_id"`
-	DefaultValue            string       `mapstructure:"default_value"`
+	Enabled      bool   `mapstructure:"enabled"`
+	HostVendorID int    `mapstructure:"host_vendor_id"`
+	DefaultValue string `mapstructure:"default_value"`
```

Comment:
I think that the vertical alignment was intentional.

Why change the indentation style of those lines ?

Reply:
@sebhtml do you mind elaborating more about  `vertical alignment was intentional` ? Changing the indentation is automatically done by the go formatter ( i am using VS stuido ) , which i believe it is easier to read.

but i am curious about other technical aspects related to it

Reply:
It may be go fmt's decision; they actually look wrong before.

Reply:
@bsardo let me know your opinion